src/freenet/support/SimpleFieldSet.java

```diff
@@ -201,7 +201,7 @@ private void read(LineReader br, int maxLength, int bufferSize, boolean utfOrIso
 				Logger.error(this, "No end marker");
 				break;
 			}
-			if ((line.length() == 0)) continue; // ignore
+			if ((line.isEmpty())) continue; // ignore
```

Comment:
So many parens here! 😀

Reply:
Fixed in ed98d8c127320b71b507f1524cce60e0d5efa356